webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/TechniqueApi.scala

```diff
@@ -481,25 +481,37 @@ class TechniqueApi(
     val restExtractor = restExtractorService
     implicit val dataName:                                                                                     String       = "techniques"
     def process0(version: ApiVersion, path: ApiPath, req: Req, params: DefaultParams, authzToken: AuthzToken): LiftResponse = {
-      case class Content(content: String)
-      def charset:          String               = RestUtils.getCharset(req)
-      implicit val decoder: JsonDecoder[Content] = DeriveJsonDecoder.gen[Content]
+      def charset: String = RestUtils.getCharset(req)
 
+      val input  = req.params.get("input").flatMap(_.map(OutputFormat.parse).headOption).getOrElse(OutputFormat.Json)
```

Comment:
It's confusing to use `OutputFormat` here, since it is also used as input. It should be `FileFormat` (or `SerializationFormat`)